buildlib/pr/main.yml

```diff
@@ -188,6 +188,11 @@ stages:
     jobs:
     - template: static_checks.yml
 
+  - stage: MAD_Tests
```

Comment:
I want to suggest something that i am trying to promote in our pipelines for some time - its not a blocker for this PR but a reminder for our self.

I would like us to use the "display name" key everywhere, the reason for this is to have consistant and unified naming convention of steps/jobs/stages etc in code and in UI.

The idea is to be able to have different naming convention for stage/job/step names one for pipeline code and one for UI display, for example:

UI should present the stage name as:
"Mad Tests"
instead of the "coding" way of:
"MAD_Tests"

And for internal pipeline use we should have it always as same convention (we should agree on one but see my suggestion below in the example as it is in AZP examples), so in this example the code should look like this:

- stage: MadTests
   displayName: Mad Tests

Reply:
Sure